installer/mac/intro.html

```diff
@@ -1,11 +1,18 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<title>Welcome to the Symphony Installer</title>
+	<title>Thanks for downloading Symphony</title>
 </head>
 <body>
 <br/>
 <br/>
-<p style="font-size: 13px; font-family: Sans-Serif; color: #000">&nbsp;&nbsp;&nbsp;We'll walk you through the steps to install Symphony</p>
+<p style="font-size: 13px; font-family: Sans-Serif; color: #000">	
+	&nbsp;&nbsp;&nbsp;Symphony allows you to:<br>
+	<ul>
+		<li>Collaborate securely</li>
+		<li>Communicate via messages, voice, and video</li>
+		<li>Connect with colleagues and coworkers</li>
+	</ul>
```

Comment:
Has no design been provided for this? I mean in terms of spacing, etc.?

Reply:
No, not for the mac installer.

Reply:
Can we ask for one?

Reply:
Spoke to the design team and they suggested that as long as the installer works with closest design to Windows, it should be fine.